contents/blog/save-engineers-time.md

```diff
@@ -0,0 +1,77 @@
+author: ["ian-vanagas"]
+categories: ["Engineering", "Product analytics"]
+---
+
+Engineers' time is valuable. Making sure it is effectively used should be a goal for both engineers and the companies they are a part of. Product Analytics can and should be helping you save time.
+
+On top of helping understand and create better products, PostHog provides the tools to maximize and save engineers time. We do this by:
+
+1.  Empowering non-engineers to access and utilize Product Analytics themselves.
+
+2.  Making it easier for engineers to implement and customize Product Analytics.
+
+To provide details about both, here are four specific ways PostHog saves engineers' time.
+
+## Test features faster
+
+The goal of implementing a new feature is for it to improve the product. Understanding when a feature has succeeded is key to doing this. You waste less time on low-impact features. Many analytics providers make testing difficult to implement or slow to get results. PostHog's testing tools help both engineers and non-engineers test and understand what features are working fast.
+
+[Feature Flags](https://posthog.com/product/feature-flags) are an example of this. Feature Flags help test new features to a subset of users, and they integrate with the rest of your product data. PostHog provides tools to non-engineers to understand Feature Flags, help set them up, and monitor their success. They have more ability to decide the "when, where, and who" of Feature Flags. For you, all you need to add is a single [if statement](https://posthog.com/manual/feature-flags#implementing-the-feature-flag) around the feature. This leaves you more time to implement the feature itself.
+
+If features fail, PostHog makes it easy to turn them off. Breaking your flow with urgent work to remove them isn't required. It just requires flipping a switch in PostHog to kill them. This is especially useful for apps that are slower to update (such as app store or extension reviews). These slow updates can take a surprising amount of time, so limiting them is critical to maximizing your useful time.
+
+As an example, the crypto wallet [Phantom](https://posthog.com/customers/phantom) uses PostHog Feature Flags as kill switches for all their features. This is because they can't deploy a new version of their Chrome Extension on demand. Feature Flags give them control of the user experience, limit feature failure, and save time.
+
+On top of Feature Flags, PostHog includes several other testing tools that are simple for you to set up and easy for non-engineers to use. They include [Session Recording](https://posthog.com/product/session-recording), [Heatmaps](https://posthog.com/product/heatmaps), [Multivariate Testing](https://posthog.com/product/experimentation-suite), and more. These create a positive feedback loop in product decisions that help features get tested faster. The team gets data on the quality of their decision and this helps them improve and speed up the next one.
```

Comment:
```suggestion
On top of Feature Flags, PostHog includes several other testing tools that are simple for you to set up and easy for non-engineers to use. They include [Session Recording](/product/session-recording), [Heatmaps](/product/heatmaps), [Multivariate Testing](/product/experimentation-suite), and more. These create a positive feedback loop in product decisions that help features get tested faster. The team gets data on the quality of their decision and this helps them improve and speed up the next one.
```
Relative paths = better. Again, this would need to be applied consistently and won't keep flagging it.